### DISCUSSION TITLE
Unable to prepare assets for download

### DISCUSSION THREAD

**Discussion by Jeh-i** [FabricMC/fabric-api#3698] [FAQ]
Hello,

Whenever i use fabric for the first 1-5 hours of reinstalling it it works fine. But after it refuses to work cause an internal error happened and its unable to prepare assets for download whatever that means.

I've tried to disable the firewall and same issue
I've tried to give the minecraft launcher administrator and same issue

Minecraft itself has no problems launching.
If i reinstall it works but that's just a Band-Aid to my original problem.

**Comment by AradMCS** [comment]
i hav the same exact problem
ig we are in restriced eareas
i am in iran
you?

**Comment by IntellsGamer** [comment]
me too

**Comment by dinhduckhoi** [comment]
me too
